internal/providers/vmware/vmware_amd64.go

```diff
@@ -97,3 +99,42 @@ func fetchRawConfig(f *resource.Fetcher) (config, error) {
 		encoding: encoding,
 	}, nil
 }
+
+func DelConfig(f *resource.Fetcher) error {
+	info := rpcvmx.NewConfig()
+
+	orig, err := info.String(GUESTINFO_USERDATA, "")
+	if err != nil {
+		return fmt.Errorf("getting config property: %w", err)
+	}
+	if orig != "" {
```

Comment:
Is there value in trying to be idempotent here by checking if it's already `e30K` and the encoding set to `base64`?